cornice/errors.py

```diff
@@ -16,6 +16,11 @@ def __init__(self, status=400):
 
     def add(self, location, name=None, description=None, **kw):
         """Registers a new error."""
+        allowed = ('body', 'querystring', 'url', 'header', 'path',
+                   'cookies', 'method')
+        if location != '' and location not in allowed:
```

Comment:
That's what colander gives when you raise a validation error that is not related to a specific node (eg. `provide param either via header or querystring`)